config/index.ts

```diff
@@ -81,13 +81,38 @@ export function writeConfig(options: WriteConfigOptions = {}): void {
   }
 }
 
+export function getRootOrDeprecatedConfigPath(
```

Comment:
Not a blocker, but is there a better term for the new config than "root"? I realize that I'm probably the one who initially started using "root", but it actually might be slightly misleading. The location where we're actually placing the config file is in the user's home directory (i.e. `~/`). Just throwing out some other options:
- Central config
- Single config
- hidden config

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/c14b1aa085ea0995951a0a73da3599756b91415b